pytorch_lightning/trainer/connectors/logger_connector/epoch_result_store.py

```diff
@@ -12,35 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from collections import defaultdict
-from typing import Any, Dict, List, Optional, Union
+from typing import Any, Dict, List, Optional
 
 import torch
 
 from pytorch_lightning.core.step_result import Result
+from pytorch_lightning.trainer.states import RunningStage
 from pytorch_lightning.utilities import DistributedType, LightningEnum
 
 
-class LoggerStages(LightningEnum):
```

Comment:
maybe move this deprecation and related changes to a separate PR?

Reply:
Fine !